src/models/Show.ts

```diff
@@ -50,22 +50,24 @@ export default class Show extends Serializable<Show> {
    * the flow based on it's continuities in stuntSheetDot.cachedFlow.
    */
   generateFlows(stuntSheetIndex: number): void {
-    if (stuntSheetIndex < 0 || stuntSheetIndex + 1 >= this.stuntSheets.length) {
```

Comment:
Just a sanity check: `>=`?

Reply:
This is correct, I want the last `stuntSheet` to be able to generate flows... The conditional looks weird though! I'll change it to `stuntSheetIndex >= this.stuntSheets.length`. 

This is in case the user decides to make the last stuntsheet have a continuity with "FMHS 8 E", "[MTHS E]", or similar where a next stuntsheet is not required.

Reply:
Got it; `stuntSheetIndex >= this.stuntSheets.length` will be a little easier to reason through, so I think that is a good idea.